package.json

```diff
@@ -56,6 +56,7 @@
     "socket.io": "^1.4.4",
     "tab-strip": "^2.3.0",
     "tv4": "^1.2.7",
+    "unzip": "^0.1.11",
```

Comment:
We already have `yauzl` (which stands for "yet another unzip library"), would be better to reuse it rather than add a dependency on another ZIP lib.